specs/UnhandledKeyPressed.md

```diff
@@ -0,0 +1,234 @@
+
+# Background
+Consumers of the old [WebBrowser](https://learn.microsoft.com/en-us/dotnet/api/system.windows.controls.webbrowser?view=windowsdesktop-7.0) control that relied on the [OnKeyDown](https://learn.microsoft.com/previous-versions/aa752133(v=vs.85)) API that allowed them to receive and handle key events not handled by the browser, [requested](https://github.com/MicrosoftEdge/WebViewFeedback/issues/468) same ability in WebView2. 
```

Comment:
The OnKeyDown links to IWebBrowser2::Navigate. Is it supposed to link to an OnKeyDown page?